src/assets/i18n/en.json

```diff
@@ -0,0 +1,226 @@
+{
+  "abnormalities":"Abnormalities",
+  "abnormalityDelete":"Delete abnormality",
+  "abnormalityDeleteQuestion":"Do you really want to delete this abnormality ?",
```

Comment:
En anglais c'est une erreur d'avoir un espace entre la fin de la phrase et sa ponctuation:
* https://english.stackexchange.com/questions/4645/is-it-ever-correct-to-have-a-space-before-a-question-or-exclamation-mark